janusgraph-core/src/main/java/org/janusgraph/diskstorage/Backend.java

```diff
@@ -85,8 +85,6 @@
     public static final String EDGESTORE_NAME = "edgestore";
     public static final String INDEXSTORE_NAME = "graphindex";
 
-    public static final String ID_STORE_NAME = "janusgraph_ids";
```

Comment:
Can we add a comment telling why ID_STORE_NAME is missing here and moved to other place? e..g To make Make ID store name configurable